File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/parking/GetParkingZoneRepsonse.java
package com.lattis.ellipse.data.network.model.response.parking;

import com.google.gson.annotations.SerializedName;
import com.lattis.ellipse.data.network.model.response.AbstractApiResponse;

import java.util.List;

/**
 * Created by lattis on 24/05/17.
 */

public class GetParkingZoneRepsonse extends AbstractApiResponse {

    @SerializedName("payload")
    GetParkingZonePayloadResponse getParkingZonePayloadResponse;

    public List<GetParkingZoneDataResponse> getParkingZoneResponse() {
        return getParkingZonePayloadResponse.getParkingZoneDataResponses();
    }

    @Override
    public String toString() {
        return "GetParkingZoneRepsonse{" +
                "GetParkingZoneRepsonse=" + getParkingZonePayloadResponse +
                '}';
    }

}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/response/parking/GetParkingZoneRepsonse.kt
package com.lattis.data.entity.response.parking

import com.google.gson.annotations.SerializedName
import com.lattis.data.entity.response.base.AbstractApiResponse

class GetParkingZoneRepsonse : AbstractApiResponse() {
    @SerializedName("payload")
    var getParkingZonePayloadResponse: GetParkingZonePayloadResponse? = null
    fun getParkingZoneDataResponse(): List<GetParkingZoneDataResponse>?{
        if(getParkingZonePayloadResponse!=null && getParkingZonePayloadResponse?.parkingZoneDataResponses!=null){
            return getParkingZonePayloadResponse!!.parkingZoneDataResponses!!
        }else{
            return null
        }
    }

    override fun toString(): String {
        return "GetParkingZoneRepsonse{" +
                "GetParkingZoneRepsonse=" + getParkingZonePayloadResponse +
                '}'
    }
}